addons/website_forum/tests/test_performance.py

```diff
@@ -0,0 +1,71 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import random
+
+from odoo.addons.website.tests.test_performance import UtilPerf
+
+
+class TestForumPerformance(UtilPerf):
```

Comment:
Also don't forget to flush to be sure we don't have hanging tracking / ... that could add either non deterministic either addon-dependent queries